Add k-notification docs (#1218)
kytos/kytos: docs/developer/components.rst

**uuid4**: Return a uuid string

=====================
Misc | k-notification

> =====================

Reviewer: I think we should also have an image here, right?

> Component representing a notification item that create and display a new item
> in the ``k-tabs`` Notifications area.

Reviewer: ```suggestion
Component representing a notification item that will be displayed
in the ``k-tabs`` Notifications area.
```
